pkg/mons/restore_quorum.go

```diff
@@ -277,15 +277,12 @@ func getMonDetails(goodMon string, monEndpoints []string) ([]string, string, str
 }
 
 func promptToContinueOrCancel(answer string) (string, error) {
-	var ROOK_PLUGIN_SKIP_PROMPTS string
-	_, ok := os.LookupEnv(ROOK_PLUGIN_SKIP_PROMPTS)
-	if ok {
+	_, ok := os.LookupEnv("ROOK_PLUGIN_SKIP_PROMPTS")
```

Comment:
Instead of just seeing if the env var exists, let's check if it also equals `true`.

Reply:
done